tests/test_mbed_targets.py

```diff
@@ -57,12 +53,24 @@ def test_iterator(self, mocked_get_target_data):
         mbed_targets = MbedTargets()
 
         mbed_target_list = [mbed_target._target_entry for mbed_target in mbed_targets]
-        self.assertEqual(fake_target_data, mbed_target_list, "The list comprehension should match the fake data")
+        self.assertEqual(
+            fake_target_data,
+            mbed_target_list,
+            "The list comprehension should match the fake data",
+        )
 
         # Check the iteration is repeatable
         mbed_target_list = [mbed_target._target_entry for mbed_target in mbed_targets]
-        self.assertEqual(fake_target_data, mbed_target_list, "The list comprehension should match the fake data")
+        self.assertEqual(
+            fake_target_data,
+            mbed_target_list,
+            "The list comprehension should match the fake data",
+        )
 
         # Iterate through the list checking the value returned matched the enumerated count
         for count, target in enumerate(mbed_targets):
-            self.assertEqual(count, target._target_entry["count"], "Iterator count values should match")
+            self.assertEqual(
```

Comment:
I'm struggling to understand what this tests. :)

Reply:
I didn't change any of the tests in this file, I just ran black on the code. I will look into this when we define the outward facing API, I suggest this is out of scope for this particular PR.

Reply:
Agreed.